pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -96,32 +134,39 @@ func (q *QueryReconciler) collectInfoFromVdb(ctx context.Context) (ctrl.Result,
 				return e
 			}
 		}
-		// extract out the communal and config information to pass down to the vclusterops API.
-		opts = append(opts,
-			restorepointsquery.WithCommunalPath(vdb.GetCommunalPath()),
-			restorepointsquery.WithConfigurationParams(q.ConfigurationParams.GetMap()),
-		)
-
 		return nil
 	})
 
 	return res, err
 }
 
-// runListRestorePoints will update the status condition before and after calling
-// list restore points api
-// Temporarily, runListRestorePoints will not call the ListRestorePoints API
-// since the dispatcher is not set up yet
-func (q *QueryReconciler) runListRestorePoints(ctx context.Context, _ *ctrl.Request) error {
+// runShowRestorePoints will update the status condition and state before and after calling
+// showrestorepoints vclusterops api
```

Comment:
This could be updated.
```suggestion
// runShowRestorePoints call the vclusterOps API to get the restore points.
```